app/services/hyrax/workflow/deposited_notification.rb

```diff
@@ -14,7 +14,7 @@ def message
       end
 
       def users_to_notify
-        user_key = Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: work_id).depositor
+        user_key = Hyrax::ValkyrieCanCanAdapter.find(nil, work_id).depositor
```

Comment:
does this need to be the alt id at all? or should it just be `Hyrax.query_service.find_by(id: work_id)`?

Reply:
turns out there's already a model resolver available to us via `@entity`. https://github.com/samvera/hyrax/pull/5120 uses that.